packages/ui5-middleware-onelogin/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "ui5-middleware-onelogin",
+  "version": "1.0.1",
+  "description": "A universal login provider for UI5 tooling",
+  "main": "/lib/index.js",
+  "scripts": {
+    "test": "npx playwright test"
+  },
+  "keywords": [
+    "UI5-tooling",
+    "SAPUI5",
+    "OPENUI5",
+    "Fiori"
+  ],
+  "repository": {
+    "url": "https://github.com/uxkjaer/ui5-middleware-onelogin.git"
+  },
+  "author": "Jakob Kjaer",
+  "license": "MIT",
+  "dependencies": {
+    "@ui5/logger": "^2.0.1",
+    "async-prompt": "*",
```

Comment:
Any reason for the asterisk, what if a major version is breaking?

Reply:
adapted to current version